internal/pkg/service/stream/definition/key/branch.go

```diff
@@ -16,7 +15,7 @@ type BranchKey struct {
 }
 
 func (v BranchKey) String() string {
-	return fmt.Sprintf("%s/%s", v.ProjectID.String(), v.BranchID.String())
+	return strconv.Itoa(int(v.ProjectID)) + "/" + v.BranchID.String()
 }
```

Comment:
It has measurable impact on http source performance.

Reply:
Usually, it doesn't need to be solved, but at ~100,000 calls/s, it has some impact.